src/components/formik/FormSuccess.module.css

```diff
@@ -0,0 +1,30 @@
+  body {
```

Comment:
By using `body`, `p`, and `i` as selectors, you have made all `body`, `p`, and `i` follow the style in this document. E.g. if I try changing `background: #EBF0F5` to `background: red` below, it will make the background red for all pages. If you want to style the whole document, please put these styles in a separate file. If you want to just style the submit page, these selectors probably have to be classes.

Reply:
There seem to be some bugs introduced by this as well, e.g. the error message should not be centered, the submit button should not be centered

<img width="1269" alt="image" src="https://user-images.githubusercontent.com/41408654/141026163-97662459-8104-40e7-a889-9533928de7a1.png">

<img width="1269" alt="image" src="https://user-images.githubusercontent.com/41408654/141026266-2a3d8ea3-d1da-4ea7-ac6b-da95faf9d348.png">